open-craft/edx-theme
# Cloudera Eucalyptus theme update

Updates the [Appsembler cloudera-dogwood theme](https://github.com/appsembler/edx-theme/tree/appsembler/cloudera-dogwood) for use with Eucalyptus.

**Sandbox URL**:

* LMS: https://eucalyptus-theme-updates.stage.opencraft.hosting
* CMS: https://studio-eucalyptus-theme-updates.stage.opencraft.hosting

**Testing instructions**:

1. Create a student account on the [ondemand.cloudera.com](https://ondemand.cloudera.com/) site, and register for one of the Free, Self-Paced courses on offer.
1. Compare the theme deployed to the [sandbox](https://eucalyptus-theme-updates.stage.opencraft.hosting) with the theme on [ondemand.cloudera.com](https://ondemand.cloudera.com/) to ensure there are major irregularities.

**Author notes and concerns**:

1. Please be aware that [ondemand.cloudera.com](https://ondemand.cloudera.com/) is the public customer site, so be careful if posting to discussion boards.
1. We haven't moved all the custom templates into standard edX locations; we've only updated the existing files to work with Eucalyptus.  So some Eucalyptus features (e.g. Bookmarks) may be missing entirely.
1. I've enabled [edX Search](http://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/edx_search.html) on the sandbox instance, but haven't seen the search indexes update properly yet.  So course searches from the Dashboard return 0 results.

**Reviewers**
